src/time_domain_reduction/time_domain_reduction.jl

```diff
@@ -866,30 +885,30 @@ function cluster_inputs(inpath, settings_path, mysetup, stage_id=-99, v=false)
                             Rep_Period_Index = [a for a in A])
 
     # Get Symbol-version of column names by type for later analysis
-    LoadCols = [Symbol("Load_MW_z"*string(i)) for i in 1:length(load_col_names) ]
+    DemandCols = Symbol.(demand_col_names)
     VarCols = [Symbol(var_col_names[i]) for i in 1:length(var_col_names) ]
     FuelCols = [Symbol(fuel_col_names[i]) for i in 1:length(fuel_col_names) ]
     ConstCol_Syms = [Symbol(ConstCols[i]) for i in 1:length(ConstCols) ]
```

Comment:
These other lines can also be simplified in the same way, but this DemandCols change avoids pulling "Load_MW_z" another time.